main.js

```diff
@@ -258,6 +258,9 @@
         });
 
         $('#clear-search').click(clearSearch);
+        $('form.search-container').submit(function(e) {
```

Comment:
An alternative to capturing the submit event here and ignoring it could be to just change the `form` element to a `div` element, since we're not using the traditional form behavior anyways. That might even be a cleaner approach... but it's no big deal either way.